ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/AddressMapper.java

```diff
@@ -21,6 +21,10 @@ public CCDAddress to(Address address) {
 
     @Override
     public Address from(CCDAddress address) {
+        if (address == null) {
```

Comment:
I still don't understand why this can be null?

Reply:
correspondence address is not mandatory in claim's Party class even and CCDParty has common class CCDAddress for Correspondance and Individual address